python/grass/jupyter/display.py

```diff
@@ -92,7 +92,10 @@ def __init__(
         self._env["GRASS_LEGEND_FILE"] = str(self._legend_file)
 
     def run(self, module, **kwargs):
-        """Run modules from "d." GRASS library"""
+        """Run modules from "d." GRASS library using grass.script.run_command().
```

Comment:
`using grass.script.run_command()` is relevant in the detailed description because it, for example, influences what syntax should be used. However, "using run_command" is not the purpose of the this function which is what this first sentence of documentaion should describe.

GRASS library is confusing: C library? Python library? Library of modules makes sense, but it is rarely used. No clear rule here. Things such as "display modules" and "modules from the display family" are used. "d-modules"/"d-commands" is not, but seemed okay to me lately, but not readable for novices similarly to the more traditional "`d.*` commands".